src/apps/app/app.element.ts

```diff
@@ -156,17 +164,15 @@ export class UmbAppElement extends UmbLitElement {
 			case RuntimeLevelModel.RUN: {
 				const pathname = pathWithoutBasePath({ start: true, end: false });
 
-				// If we are on the installer or upgrade page, redirect to the root
-				// but if not, keep the current path but replace state anyway to initialize the router
-				let currentRoute = location.href;
-				const savedRoute = sessionStorage.getItem('umb:auth:redirect');
-				if (savedRoute) {
-					sessionStorage.removeItem('umb:auth:redirect');
-					currentRoute = savedRoute;
+				// If we are on installer or upgrade page, redirect to the root since we are in the RUN state
+				if (pathname === '/install' || pathname === '/upgrade') {
```

Comment:
Is there any way we can have constants for the install and upgrade path names?

Reply:
Good idea, although they are only used in this file (where the main router is registered). I would like to find some way to better solidify the fact that all "known routes" should be ignored specifically here, so would be good to have a more programmatic approach to that in the future.